data/categories.txt

```diff
@@ -6491,6 +6491,30 @@ el:μπάνιο|χώρος ανάγκης|αποχωρητήριο|wc
 sk:3Toalety|wc
 mr:शौचालय
 
+toilets-yes|@toilet
```

Comment:
No need a separate entry. Fix an existing one:
amenity-toilets|toilets-yes|@toilet

Reply:
Thanks! I didn't want to duplicate but I didn't know how to do it. I knew in `type_strings.txt` you could use the `ref =` but I didn't know how to do it in `categories.txt`